KOService.Application/Commands/Repair/CreateRepairCommand.cs

```diff
@@ -0,0 +1,20 @@
+﻿using MediatR;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace KOService.Application.Commands.Repair
+{
+    public class CreateRepairCommand : IRequest
+    {
+        public CreateRepairCommand()
+        {
+            Id = Guid.NewGuid();
```

Comment:
Jak EmployeeTask dziedziczy po Entity to trzeba tak robić?

Reply:
Dobrze, że Bartek jesteś czujny. Ja zrobiłem ten nowy guid w tym commandzie zeby potem sobie zwrocic go w responsie i wtedy na froncie od razu można sobie go pobrać, co może się przydać, więc chyba lepiej usunąć ten .NewGuid() z entity i wtedy każdy command bedzie robil .NewGuid() (to można stworzyć bazową klasę command do tego, żeby nie zapominać i nie pisać za każdym razem). Co myślicie ?